### ISSUE TITLE
Know when the grid is ready ?

### ISSUE THREAD

**Issue by yveslange** [vaadin/vaadin-grid#1068]
Is it possible to know when the columns (and/or rows) of the grid are ready ? 

My goal is to display a "loader" while the grid is generating the columns (and/or rows) and is not ready to be used.

**Comment by YuriyVaadin** [comment]
Dear @yveslange,

Unfortunately, we do not have a way to find out when the columns (and/or rows) of the grid are ready. Could you please provide us with the information on your use-case, probably we will be able to find the reason of slow loading and improve performance.

Best regards,
Elements Support

**Comment by yveslange** [comment]
Hi @YuriyVaadin,
the grid is slow because it has 90 columns that are stamped through a <dom-repeat> to automatically be generated.

I found that it is possible to listen to the `index` of the current displayed <dom-repeat> element. I used it to know when the columns of the grid are rendered. Thus, I can display a `loader` while the columns are loading. 

Didn't found any solution to know when the rows are rendered.
